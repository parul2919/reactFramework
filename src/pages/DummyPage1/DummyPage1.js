import React from "react";
import DummyTemplate1 from "../../components/template/DummyTemplate1/DummyTemplate1";

const wrapperStyles = {
  maxWidth: "100%",
  margin: "auto",
};

const DummyPage1 = () => (
  <div style={wrapperStyles}>
    <DummyTemplate1 />
  </div>
);

export default DummyPage1;
