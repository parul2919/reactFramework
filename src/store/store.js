import { applyMiddleware, createStore } from "redux";
// Logger with default options
// import logger from 'redux-logger';
import createSagaMiddleware from "redux-saga";
import rootReducer from "./reducers/rootReducer";
import rootSaga from "./sagas/rootSaga";

const sagaMiddleware = createSagaMiddleware();
const store = createStore(
  rootReducer,
  applyMiddleware(sagaMiddleware)
  // applyMiddleware(sagaMiddleware, logger),
);
sagaMiddleware.run(rootSaga);

export default store;
