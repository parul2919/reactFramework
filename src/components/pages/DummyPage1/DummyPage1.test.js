import React from "react";
import { render } from "@testing-library/react";
import { BrowserRouter as Router } from "react-router-dom";
import { Provider } from "react-redux";
import store from "../../../store/store";
import DummyPage1 from "./DummyPage1";

describe("<Component /> spec", () => {
  it("renders the component", () => {
    const view = render(
      <Provider store={store}>
        <Router>
          <DummyPage1 />
        </Router>
      </Provider>
    );
    expect(view).toMatchSnapshot();
  });
});
