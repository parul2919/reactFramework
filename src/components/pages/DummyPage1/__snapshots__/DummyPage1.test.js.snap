// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Component /> spec renders the component 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        style="max-width: 100%; margin: auto;"
      >
        <div
          class="sc-hKMtZM lbYHHX dummy-template"
        >
          <header
            class="header sc-gsnTZi bgEuUO"
          >
            <nav
              class="navbar navbar-expand"
            >
              <a
                class="navbar-brand "
                href="/"
              >
                <img
                  alt="logo-img"
                  src="test-file-stub"
                  width="150"
                />
              </a>
              <div
                class="collapse navbar-collapse"
              >
                <ul
                  class="navbar-nav ml-auto "
                >
                  <a
                    class="nav-link"
                    href="/"
                    style="text-decoration: underline;"
                    target="_self"
                  >
                    Demo Link 1
                  </a>
                  <a
                    class="nav-link"
                    href="/demoLink2"
                    style="text-decoration: none;"
                    target="_self"
                  >
                    Demo Link 2
                  </a>
                  <a
                    class="nav-link"
                    href="/demoLink3"
                    style="text-decoration: none;"
                    target="_self"
                  >
                    Demo Link 3
                  </a>
                </ul>
              </div>
            </nav>
          </header>
          <section
            class="main"
          >
            <p>
              Checking env variables value
            </p>
            <div
              class="mb-5"
            />
            <div
              class="mb-5"
            />
            <section
              class="dummy-organism sc-dkzDqf JApfA"
            >
              <div
                class="dummy-organism--text"
              >
                This article i s about artistic painting. For other uses, see Painting (disambiguation). "Painter" redirects here. For other uses, see Painter (disambiguation). The Mona Lisa (1503–1517) by Leonardo da Vinci is one of the world's most recognizable paintings. An artistic depiction of a group of rhinos was made in the Chauvet Cave 30,000 to 32,000 years ago. Painting is the practice of applying paint, pigment, color or other medium to a solid surface (called the "matrix" or "support").[1] The medium is commonly applied to the base with a brush, but other implements, such as knives, sponges, and airbrushes, can be used. In art, the term painting describes both the act and the result of the action (the final work is called "a painting"). The support for paintings includes such surfaces as walls, paper, canvas, wood, glass, lacquer, pottery, leaf, copper and concrete, and the painting may incorporate multiple other materials, including sand, clay, paper, plaster, gold leaf, and even whole objects. Painting is an important form in the visual arts, bringing in elements such as drawing, composition, gesture (as in gestural painting), narration (as in narrative art), and abstraction (as in abstract art).[2] Paintings can be naturalistic and representational (as in still life and landscape painting), photographic, abstract, narrative, symbolistic (as in Symbolist art), emotive (as in Expressionism) or political in nature (as in Artivism).
              </div>
            </section>
            <p>
              success
               ....
               is here to check saga & reducer connecttion
            </p>
          </section>
          <section
            class="footer sc-bczRLJ eRSPWO"
          >
            <nav
              style="max-width: 1440px; margin: auto;"
            >
              <a
                class="footer-logo"
                href="/"
              >
                <img
                  alt="logo-img"
                  src="test-file-stub"
                  width="150"
                />
              </a>
            </nav>
            <div
              class="disclaimer-text text-center"
            >
              Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ips um has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged.
            </div>
          </section>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      style="max-width: 100%; margin: auto;"
    >
      <div
        class="sc-hKMtZM lbYHHX dummy-template"
      >
        <header
          class="header sc-gsnTZi bgEuUO"
        >
          <nav
            class="navbar navbar-expand"
          >
            <a
              class="navbar-brand "
              href="/"
            >
              <img
                alt="logo-img"
                src="test-file-stub"
                width="150"
              />
            </a>
            <div
              class="collapse navbar-collapse"
            >
              <ul
                class="navbar-nav ml-auto "
              >
                <a
                  class="nav-link"
                  href="/"
                  style="text-decoration: underline;"
                  target="_self"
                >
                  Demo Link 1
                </a>
                <a
                  class="nav-link"
                  href="/demoLink2"
                  style="text-decoration: none;"
                  target="_self"
                >
                  Demo Link 2
                </a>
                <a
                  class="nav-link"
                  href="/demoLink3"
                  style="text-decoration: none;"
                  target="_self"
                >
                  Demo Link 3
                </a>
              </ul>
            </div>
          </nav>
        </header>
        <section
          class="main"
        >
          <p>
            Checking env variables value
          </p>
          <div
            class="mb-5"
          />
          <div
            class="mb-5"
          />
          <section
            class="dummy-organism sc-dkzDqf JApfA"
          >
            <div
              class="dummy-organism--text"
            >
              This article i s about artistic painting. For other uses, see Painting (disambiguation). "Painter" redirects here. For other uses, see Painter (disambiguation). The Mona Lisa (1503–1517) by Leonardo da Vinci is one of the world's most recognizable paintings. An artistic depiction of a group of rhinos was made in the Chauvet Cave 30,000 to 32,000 years ago. Painting is the practice of applying paint, pigment, color or other medium to a solid surface (called the "matrix" or "support").[1] The medium is commonly applied to the base with a brush, but other implements, such as knives, sponges, and airbrushes, can be used. In art, the term painting describes both the act and the result of the action (the final work is called "a painting"). The support for paintings includes such surfaces as walls, paper, canvas, wood, glass, lacquer, pottery, leaf, copper and concrete, and the painting may incorporate multiple other materials, including sand, clay, paper, plaster, gold leaf, and even whole objects. Painting is an important form in the visual arts, bringing in elements such as drawing, composition, gesture (as in gestural painting), narration (as in narrative art), and abstraction (as in abstract art).[2] Paintings can be naturalistic and representational (as in still life and landscape painting), photographic, abstract, narrative, symbolistic (as in Symbolist art), emotive (as in Expressionism) or political in nature (as in Artivism).
            </div>
          </section>
          <p>
            success
             ....
             is here to check saga & reducer connecttion
          </p>
        </section>
        <section
          class="footer sc-bczRLJ eRSPWO"
        >
          <nav
            style="max-width: 1440px; margin: auto;"
          >
            <a
              class="footer-logo"
              href="/"
            >
              <img
                alt="logo-img"
                src="test-file-stub"
                width="150"
              />
            </a>
          </nav>
          <div
            class="disclaimer-text text-center"
          >
            Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ips um has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged.
          </div>
        </section>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
